examples/cross_canister_calls/canisters/canister1/canister1.ts

```diff
@@ -30,28 +30,30 @@ let canister2 = ic.canisters.Canister2<Canister2>(
     Principal.fromText('ryjl3-tyaaa-aaaaa-aaaba-cai')
 );
 
-export function* transfer(
+export async function transfer(
```

Comment:
Can you undo these changes for now? This will break the tests and I'd like to go and fix everything in one fell swoop.